zcash_primitives/src/transaction/mod.rs

```diff
@@ -371,6 +362,54 @@ impl PartialEq for Transaction {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum OrchardBundle<A: orchard::bundle::Authorization> {
+    OrchardVanilla(Box<Bundle<A, Amount, OrchardVanilla>>),
+    #[cfg(zcash_unstable = "nu6" /* TODO nu7 */ )]
+    OrchardZSA(Box<Bundle<A, Amount, OrchardZSA>>),
+}
```

Comment:
```rust
pub enum OrchardBundle<A: orchard::bundle::Authorization> {
    OrchardVanilla(Bundle<A, Amount, OrchardVanilla>),
    // #[cfg(zcash_unstable = "nu6" /* TODO nu7 */ )]
    OrchardZSA(Bundle<A, Amount, OrchardZSA>),
}
```

Reply:
Not sure what you mean by //. Can you elaborate?